src/index.html

```diff
@@ -21,7 +21,430 @@
+          5K Monoblock (MNED2UA/A)
+        </div>
+
+        <div class="card__code">Product code: 195434</div>
+
+        <div class="card__rating">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <div class="card__reviews">Reviews: 5</div>
+        </div>
+
+        <div class="card__price">
+          <div class="card__price-name">Price:</div>
+          <div class="card__price-amount">$2,199</div>
+        </div>
+
+        <a
+          class="linkbtn card__buy-button"
+          href="#"
+          data-qa="card-hover"
+        >
+          buy
+        </a>
+      </div>
+
+      <div class="card">
+        <div class="card__imageframe">
+          <img
+            alt="image"
+            class="card__image"
+            src="./images/imac.jpeg"
+          />
+        </div>
+
+        <div class="card__title">
+          APPLE A1419 iMac 27" Retina
+          <br />
+          5K Monoblock (MNED2UA/A)
+        </div>
+
+        <div class="card__code">Product code: 195434</div>
+
+        <div class="card__rating">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <div class="card__reviews">Reviews: 5</div>
+        </div>
+
+        <div class="card__price">
+          <div class="card__price-name">Price:</div>
+          <div class="card__price-amount">$2,199</div>
+        </div>
+
+        <a
+          class="linkbtn card__buy-button"
+          href="#"
+        >
+          buy
+        </a>
+      </div>
+
+      <div class="card">
+        <div class="card__imageframe">
+          <img
+            alt="image"
+            class="card__image"
+            src="./images/imac.jpeg"
+          />
+        </div>
+
+        <div class="card__title">
+          APPLE A1419 iMac 27" Retina
+          <br />
+          5K Monoblock (MNED2UA/A)
+        </div>
+
+        <div class="card__code">Product code: 195434</div>
+
+        <div class="card__rating">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <div class="card__reviews">Reviews: 5</div>
+        </div>
+
+        <div class="card__price">
+          <div class="card__price-name">Price:</div>
+          <div class="card__price-amount">$2,199</div>
+        </div>
+
+        <a
+          class="linkbtn card__buy-button"
+          href="#"
```

Comment:
The 'href' attribute is set to '#', which does not lead to any specific page. Consider updating it to point to the correct URL or page.